datacenterlight/locale/de/LC_MESSAGES/django.po

```diff
@@ -329,6 +329,14 @@ msgstr "wird an der Kasse angewendet"
 msgid "Credit Card"
 msgstr "Kreditkarte"
 
+msgid ""
+"Please select one of the cards that you used before or fill in your credit "
+"card information below. We are using <a href=\"https://stripe.com\" target="
+"\"_blank\">Stripe</a> for payment and do not store your information in our "
+"database."
+msgstr ""
+""
```

Comment:
Bitte wähle eine der zuvor genutzten Kreditkarten oder gib deine Kreditkartendetails unten an. Die Bezahlung wird über <a href=\"https://stripe.com\" target="\"_blank\">Stripe</a> abgewickelt. Wir speichern deine Kreditkartendetails nicht in unserer Datenbank.

Reply:
In my sense "dein" should be lowercase, but I have made it uppercase because of the consistency with other messages.